### PR TITLE
New Emoji Dictionary Added

### PR DESCRIPTION
**Pull request by saurabhshadow**

### CODE REVIEW

File: services/coolifyService.spec.js [hunk 1]
```diff
@@ -64,8 +64,15 @@ describe('coolifyService.alphaNumericeName', () => {
   it('should replace with doublestruck letter', () => {
     expect(alphaNumericeName('hello')[13]).to.equal('𝕙𝕖𝕝𝕝𝕠');
   });
-  
-  it('should replace with unicodemix letter', () => {
+
+it('should replace with unicodemix letter', () => {
```
Comment (mohitkyadav) [inline services/coolifyService.spec.js]: Fix indentation

Comment (saurabhshadow) [reply services/coolifyService.spec.js]: fixed!

**Comment by saurabhshadow** [comment]
I have **### fixed the indentation**, please have a look!

File: services/uncoolifyService.js [hunk 2]
```diff
@@ -367,6 +367,35 @@ uncoolifyDicts.unicodemix = {
 	'ℨ':'z',
 }
 
+uncoolifyDicts.newemoji={
+	'🥨'='a',
```
Comment (mohitkyadav) [inline services/uncoolifyService.js]: That's  not a dictionary, it should be
```
{
    'key': 'value'
}
```

Comment (saurabhshadow) [reply services/uncoolifyService.js]: > That's not a dictionary, it should be
> 
> ```
> {
>     'key': 'value'
> }
> ```

what does that mean?

Comment (mohitkyadav) [reply services/uncoolifyService.js]: Please run the app and test all corner case each time you commit.

Comment (saurabhshadow) [reply services/uncoolifyService.js]: > Please run the app and test all corner case each time you commit.

this time its all set.
please have a look!

Comment (saurabhshadow) [reply services/uncoolifyService.js]: the curly bracket should end like this?
{
 'key':'value'
};

Comment (saurabhshadow) [reply services/uncoolifyService.js]: Some of the dictionary end with 

} ;
and some do not?

Comment (saurabhshadow) [reply services/uncoolifyService.js]: Everything is checked and done properly.
Please review once!

File: services/coolifyService.js [hunk 3]
```diff
@@ -362,6 +362,35 @@ coolifyDicts.unicodemix = {
...
+	'i':'🔩',
+	'j':'🃏',
+	'k':'🔀',
+	'l':'🛴',
+	'm':'Ⓜ️',
+	'n':'♑️',
+	'o':'🌍',
+	'p':'🅿️',
+	'q':'🍭',
+	'r':'🎵',
```
Comment (mohitkyadav) [inline services/coolifyService.js]: This looks more like `n`.

File: services/coolifyService.js [hunk 4]
```diff
@@ -362,6 +362,35 @@ coolifyDicts.unicodemix = {
 	'z':'ℨ',
 }
 
+coolifyDicts.newemoji= {
+	'a':'🥨',
+	'b':'🅱',
+	'c':'↪️',
+	'd':'👂🏼',
```
Comment (mohitkyadav) [inline services/coolifyService.js]: This looks more like `p`.

File: services/coolifyService.js (hunk 3 above)
Comment (saurabhshadow) [reply services/coolifyService.js]: okk, I'll change!

File: services/coolifyService.js (hunk 4 above)
Comment (saurabhshadow) [reply services/coolifyService.js]: okk...I'll change!
